package.json

```diff
@@ -90,7 +90,6 @@
     "@typescript-eslint/eslint-plugin": "^3.1.0",
     "@typescript-eslint/parser": "^3.1.0",
     "babel-plugin-rewire-ts": "^1.3.3",
-    "casperjs": "casperjs/casperjs",
```

Comment:
This commits needs a provision version bump because we are removing a dependency.

Reply:
Forgot about that, thanks, bumped the version now.